### ISSUE TITLE
force:cmdt:record:create gives `Error: Unexpected arguments` for field key-value pair args in 7.184.2

### ISSUE THREAD

**Issue by chrishnicholas** [forcedotcom/cli#1893]
### Summary

A `sfdx force:cmdt:record:create` command that worked in previous versions of sfdx-cli seems to have stopped working in v7.184.2.

### Steps To Reproduce:

1.  Use sfdx version: `sfdx-cli/7.184.2 darwin-x64 node-v18.12.1`
1. Run command like:
```
sfdx force:cmdt:record:create -t CustomMetaType1__mdt -n CUSTOM_RECORD_NAME Custom_Field1__c="ABC" Custom_Field2__c="DEF"
```

### Expected result

Metadata record is created.

### Actual result

Receive error message:
```
Error: Unexpected arguments: Custom_Field1__c=ABC, Custom_Field2__c=DEF
```

### System Information

- Which shell/terminal are you using? (e.g. bash, zsh, powershell 5, powershell 7, cmd.exe, etc.)
```
{
  "cliVersion": "sfdx-cli/7.184.2",
  "architecture": "darwin-x64",
  "nodeVersion": "node-v18.12.1",
  "pluginVersions": [
    "@oclif/plugin-autocomplete 1.3.8 (core)",
    "@oclif/plugin-commands 2.2.2 (core)",
    "@oclif/plugin-help 5.1.20 (core)",
    "@oclif/plugin-not-found 2.3.13 (core)",
    "@oclif/plugin-plugins 2.1.8 (core)",
    "@oclif/plugin-search 0.0.6 (core)",
    "@oclif/plugin-update 3.0.9 (core)",
    "@oclif/plugin-version 1.1.4 (core)",
    "@oclif/plugin-warn-if-update-available 2.0.18 (core)",
    "@oclif/plugin-which 2.2.6 (core)",
    "@salesforce/lwc-dev-server 2.11.0 (user)",
    "alias 2.1.16 (core)",
    "apex 1.3.0 (core)",
    "auth 2.3.10 (core)",
    "community 2.1.3 (core)",
    "config 1.4.23 (core)",
    "custom-metadata 2.0.11 (core)",
    "data 2.1.22 (core)",
    "generator 2.0.15 (core)",
    "info 2.3.2 (core)",
    "limits 2.2.2 (core)",
    "org 2.2.22 (core)",
    "packaging 1.12.2 (core)",
    "schema 2.2.2 (core)",
    "signups 1.2.12 (core)",
    "source 2.3.12 (core)",
    "telemetry 2.0.5 (core)",
    "templates 55.1.0 (core)",
    "trust 2.2.6 (core)",
    "user 2.1.25 (core)",
    "@salesforce/sfdx-plugin-lwc-test 1.0.1 (core)",
    "salesforce-alm 54.8.5 (core)"
  ],
  "osVersion": "Darwin 22.2.0",
  "shell": "zsh",
  "rootPath": "/Users/christopher.nicholas/.local/share/sfdx/client/7.184.2-8a80929"
}
```
### Additional information

None.

**Comment by peternhale** [comment]
@chrishnicholas Thank you fro reporting this issue and our apologies for the inconvenience. The issue has been fixed and if you would like to give it a try you can install the plugin directly with `sfdx plugins:install @salesforce/plugin-custom-metadata`

**Comment by chrishnicholas** [comment]
Thanks for the quick turnaround on this one and the workaround (to install the updated plugin). Out of interest, how long does it typically take for a plugin to become the default version shipped with the cli?

**Comment by peternhale** [comment]
@chrishnicholas the fix will be available in tomorrow's sfdx release candidate and will be promoted to latest a week from tomorrow.

**Comment by shetzel** [comment]
The fix is in the current production CLI, v7.186.2
